project.go

```diff
@@ -0,0 +1,392 @@
+		Name: "project-remove",
+		Desc: "removes the given project",
+	}
+}
+
+func (c *projectRemove) Run(ctx *cmd.Context, client *cmd.Client) error {
+	if c.name == "" {
+		return errors.New("please provide the name of the project")
+	}
+	config, err := loadConfigFile()
+	if err != nil {
+		return errors.New("unable to load environments file, please make sure that tranor is properly configured")
+	}
+	if !c.Confirm(ctx, fmt.Sprintf("Are you sure you want to remove the project %q?", c.name)) {
+		return nil
+	}
+	envNames := config.envNames()
+	appNames := make([]string, len(envNames))
+	for i, envName := range envNames {
+		appNames[i] = fmt.Sprintf("%s-%s", c.name, envName)
+	}
+	errs, err := deleteApps(appNames, client)
+	if err != nil {
+		return err
+	}
+	var notFound int
+	for _, err := range errs {
+		if err != nil {
+			if e, ok := err.(*tsuruerrors.HTTP); ok && e.Code == http.StatusNotFound {
+				notFound++
+			}
+		}
+	}
+	if notFound == len(errs) {
+		return errors.New("project not found")
+	}
+	fmt.Fprintln(ctx.Stdout, "Project successfully removed!")
+	return nil
+}
+
+func (c *projectRemove) Flags() *gnuflag.FlagSet {
+	if c.fs == nil {
+		c.fs = c.ConfirmationCommand.Flags()
+		c.fs.StringVar(&c.name, "name", "", "name of the project to b remove")
+		c.fs.StringVar(&c.name, "n", "", "name of the project to remove")
+	}
+	return c.fs
+}
+
+type projectInfo struct {
+	name string
+	fs   *gnuflag.FlagSet
+}
+
+func (c *projectInfo) Info() *cmd.Info {
+	return &cmd.Info{
+		Name: "project-info",
+	}
+}
+
+func (c *projectInfo) Run(ctx *cmd.Context, client *cmd.Client) error {
+	if c.name == "" {
+		return errors.New("please provide the name of the project")
+	}
+	apps, err := c.projectApps(client)
+	if err != nil {
+		return err
+	}
+	fmt.Fprintf(ctx.Stdout, "Project name: %s\n\n", c.name)
+	var envs cmd.Table
+	envs.Headers = cmd.Row{"Environment", "Address", "Version", "Deploy date"}
+	for _, app := range apps {
+		row := cmd.Row{app.Env.Name, app.Addr, "", ""}
+		if deploy, err := lastDeploy(client, app.Name); err == nil && deploy.Image != "" {
+			row[2] = deploy.Image
+			row[3] = deploy.Timestamp.Format(time.RFC1123)
+		}
+		envs.AddRow(row)
+	}
+	ctx.Stdout.Write(envs.Bytes())
+	return nil
+}
+
+func (c *projectInfo) projectApps(client *cmd.Client) ([]app, error) {
+	config, err := loadConfigFile()
+	if err != nil {
+		return nil, errors.New("unable to load environments file, please make sure that tranor is properly configured")
+	}
+	apps, err := listApps(client, map[string]string{"name": c.name})
+	if err != nil {
+		return nil, err
+	}
+	var projectApps []app
+	for _, env := range config.Environments {
+		for _, app := range apps {
+			if len(app.CName) != 1 {
+				continue
+			}
+			nameParts := env.nameRegexp().FindStringSubmatch(app.Name)
+			dnsParts := env.dnsRegexp().FindStringSubmatch(app.CName[0])
+			if len(nameParts) == 2 && len(dnsParts) == 2 && nameParts[1] == dnsParts[1] && nameParts[1] == c.name {
```

Comment:
I liked this `if` :) You are making very sure that this is really an tranor-friendly app, right?